src/common/helpers/calculate-age.helper.ts

```diff
@@ -0,0 +1,5 @@
+export const calculateAge = (birthDate: Date): number => {
+  const today = new Date();
+
+  return today.getFullYear() - birthDate.getFullYear();
```

Comment:
I understood your code, but, has a potential bug.
Try using it in this way

```suggestion
  let age = today.getFullYear() - birthDate.getFullYear();
  const monthDiff = today.getMonth() - birthDate.getMonth();
  
  if (monthDiff < 0 || (monthDiff === 0 && today.getDate() < birthDate.getDate())) {
    age--;
  }
  
  return age;
```